internal/docker/builder.go

```diff
@@ -351,11 +346,19 @@ func (d *Builder) construct(bprint b.Blueprint) (errs []error) {
 	return errs
 }
 
+func toChanges(labels map[string]string) []string {
+	changes := make([]string, 0)
```

Comment:
`var changes []string` is shorter, or `changes := make([]string, 0, len(labels))` to preallocate a capacity of the underlying array.